### PR TITLE
636: Adding more seed data to make the graphs look nicer and more mea…

### PR DESCRIPTION
**Pull request by dconey646**
…ningful

### CODE REVIEW

File: data/00400_ldu.js [hunk 1]
```diff
@@ -7,7 +7,13 @@ exports.seed = function (knex, Promise) {
...
       // Inserts seed entries
       return knex(tableName).insert([
-        { description: 'test ldu', region_id: regionId.id }
+        { description: 'test ldu 1', region_id: regionId.id },
+        { description: 'test ldu 2', region_id: regionId.id },
+        { description: 'test ldu 3', region_id: regionId.id },
+        { description: 'test ldu 4', region_id: regionId.id },
+        { description: 'test ldu 5', region_id: regionId.id },
+        { description: 'test ldu 6', region_id: regionId.id },
+        { description: 'test ldu 7', region_id: regionId.id }
```
Comment (caoimheb) [inline data/00400_ldu.js]: Only one LDU will have teams etc, so on the region graph there will only be one line? Can we add more teams to one other LDU?

File: data/00400_ldu.js [hunk 2]
```diff
@@ -7,7 +7,13 @@ exports.seed = function (knex, Promise) {
     .then(function (regionId) {
```
Comment (caoimheb) [inline data/00400_ldu.js]: Add more than one region, for the national level graph?

File: data/00400_ldu.js (hunk 1 above)
Comment (dconey646) [inline data/00400_ldu.js]: Yea, sure. Good spot.

File: data/00400_ldu.js (hunk 2 above)
Comment (dconey646) [inline data/00400_ldu.js]: 👍

File: data/00400_ldu.js [hunk 3]
```diff
@@ -2,12 +2,20 @@ var tableName = 'ldu'
 
 exports.seed = function (knex, Promise) {
   // Deletes ALL existing entries
+  var regionIds
   return knex(tableName).del()
-    .return(knex('region').select('id').first())
-    .then(function (regionId) {
+    .return(knex('region').select('id').limit(3))
+    .then(function (results) {
+      console.log('LDU ' + results)
```
Comment (caoimheb) [inline data/00400_ldu.js]: console.log

File: data/00500_team.js [hunk 4]
```diff
@@ -2,12 +2,20 @@ var tableName = 'team'
 
 exports.seed = function (knex, Promise) {
   // Deletes ALL existing entries
+  var lduIds
   return knex(tableName).del()
-    .return(knex('ldu').select('id').first())
-    .then(function (lduId) {
+    .return(knex('ldu').select('id').limit(3))
+    .then(function (results) {
+      console.log('TEAM ' + results)
```
Comment (caoimheb) [inline data/00500_team.js]: console.log

File: data/00600_workload_owner.js [hunk 5]
```diff
@@ -2,19 +2,24 @@ var tableName = 'workload_owner'
...
   // Deletes ALL existing entries
-  var teamId
+  var teamIds
   return knex(tableName).del()
-    .return(knex('team').select('id').first())
-    .then(function (id) {
-      teamId = id
+    .return(knex('team').select('id').limit(4))
+    .then(function (results) {
+      console.log('TEAM 2 ' + results)
```
Comment (caoimheb) [inline data/00600_workload_owner.js]: console.log

File: data/00600_workload_owner.js [hunk 6]
```diff
@@ -2,19 +2,24 @@ var tableName = 'workload_owner'
...
+      teamIds = results
       return knex('offender_manager').select('id')
         .limit(5)
     })
     .then(function (omIds) {
       // Inserts seed entries
+      console.log('OMids ' + omIds)
```
Comment (caoimheb) [inline data/00600_workload_owner.js]: Another one :)

File: data/00700_workload.js [hunk 7]
```diff
@@ -4,17 +4,38 @@ const workloadRow = { workload_owner_id: 1, total_cases: 15, monthly_sdrs: 10, s
...
+  var partOneWorkloads = []
+  var partTwoWorkloads = []
+  var partThreeWorkloads = []
+  var partFourWorkloads = []
   return knex(tableName).del()
     .then(function () {
       return knex('workload_owner').select('id')
     })
     .then(function (workloadOwners) {
+      console.log(workloadOwners)
```
Comment (caoimheb) [inline data/00700_workload.js]: ....

File: data/01000_workload_points_calculations.js [hunk 8]
```diff
@@ -28,6 +30,12 @@ exports.seed = function (knex, Promise) {
         workloadPointsCalculationsToInsert.push({ workload_id: workloadId.id, workload_report_id: reportId.id, workload_points_id: currentPointsId.id, effective_from: effectiveFromDate, effective_to: effectiveFromDate.getDate() + 30, total_points: Math.floor(Math.random() * 25) + 30, available_points: 190, paroms_points: 50, sdr_conversion_points: 50, sdr_points: 50, nominal_target: 0, reduction_hours: Math.floor(Math.random() * 6) + 1 })
         effectiveFromDate.setDate(effectiveFromDate.getDate() + 30)        
       }
-      return knex(tableName).insert(workloadPointsCalculationsToInsert)
+      partOneWorkloadPointsCalculations = workloadPointsCalculationsToInsert.slice(0, workloadPointsCalculationsToInsert.length / 2)
+      partTwoWorkloadPointsCalculations = workloadPointsCalculationsToInsert.slice(0, workloadPointsCalculationsToInsert.length / 2)
```
Comment (caoimheb) [inline data/01000_workload_points_calculations.js]: Inserting same set twice?

**Pull request merged**